### ISSUE TITLE
Installation on Python 3.12 (pre-release) fails

### ISSUE THREAD

**Issue by shenef** [Zuzu-Typ/PyGLM#218]
On Python 3.12.0rc3:
```
PS E:\SoS-TAS> pip install PyGLM
Collecting PyGLM
  Using cached PyGLM-2.7.0.tar.gz (4.6 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [22 lines of output]
      Traceback (most recent call last):
        File "C:\Users\shen\AppData\Local\Programs\Python\Python312\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "C:\Users\shen\AppData\Local\Programs\Python\Python312\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\shen\AppData\Local\Programs\Python\Python312\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\shen\AppData\Local\Temp\pip-build-env-x4jbfb40\overlay\Lib\site-packages\setuptools\build_meta.py", line 355, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\shen\AppData\Local\Temp\pip-build-env-x4jbfb40\overlay\Lib\site-packages\setuptools\build_meta.py", line 325, in _get_build_requires
          self.run_setup()
        File "C:\Users\shen\AppData\Local\Temp\pip-build-env-x4jbfb40\overlay\Lib\site-packages\setuptools\build_meta.py", line 507, in run_setup
          super(_BuildMetaLegacyBackend, self).run_setup(setup_script=setup_script)
        File "C:\Users\shen\AppData\Local\Temp\pip-build-env-x4jbfb40\overlay\Lib\site-packages\setuptools\build_meta.py", line 341, in run_setup
          exec(code, locals())
        File "<string>", line 33, in <module>
        File "C:\Users\shen\AppData\Local\Programs\Python\Python312\Lib\shutil.py", line 448, in copy2
          _winapi.CopyFile2(src_, dst_, flags)
      FileNotFoundError: [WinError 3] The system cannot find the path specified
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by szabolcsdombi** [comment]
The source distribution is missing files, see https://github.com/Zuzu-Typ/PyGLM/pull/219

**Comment by Zuzu-Typ** [comment]
Fixed by #219 (in release 2.7.1)
